config/proto_conversions.go

```diff
@@ -848,7 +848,7 @@ func toRDKSlice[PT, RT any](
 	for _, proto := range protoList {
 		rdk, err := toRDK(proto)
 		if err != nil {
-			logger.Debugw("error converting from proto to config", "type", reflect.TypeOf(proto).String(), "error", err)
```

Comment:
This has been a debug-level log since seemingly 2022, and my _guess_ is that it was to indicate these errors weren't that important? Or, it was a typo? In any case, I see no real argument that this should be a debug log; it represents a real error in parsing, and this `toRDKSlice` generic function is used all over the place to convert pieces of a `config.Config` to proto. @shawnbmccarthy encountered a "debug" log here showing that an API was invalid, which does not seem like a "debug" issue to me.